src/main/target/SPRACINGF4EVO/target.c

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+#include "drivers/dma.h"
+
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    { TIM9,  IO_TAG(PA3),   TIM_Channel_2, TIM_USE_PPM | TIM_USE_PWM,   0, GPIO_AF_TIM9, NULL, 0, 0 }, // PPM / PWM1 / UART2 RX
```

Comment:
DMA channels for timers are defined here:
https://github.com/cleanflight/cleanflight/blob/master/src/main/drivers/timer_def.h#L402-L405
DMA streams are here:
https://github.com/cleanflight/cleanflight/blob/master/src/main/drivers/timer_def.h#L333-L336

For Tim8Ch2, it should be 
0: DMA2_ST3/DMA_Channel_0 
1: DMA2_ST2/DMA_Channel_7

Can you check that against F4 RM ?